src/module/rules/armor.mjs

```
@@ -1,31 +1,32 @@
-    console.log("Elkan 5e  |  Initializing Armor")
+    console.log("Elkan 5e  |  Initializing Armor");
 
-    if (!armor){
-        //Delete List
-        delete CONFIG.DND5E.armorIds.ringmail
-        delete CONFIG.DND5E.armorIds.studded
-        delete CONFIG.DND5E.shieldIds.shield
+    if (!armor) {
+        // Delete List
+        ["ringmail", "studded"].forEach(id => delete CONFIG.DND5E.armorIds[id]);
+        delete CONFIG.DND5E.shieldIds.shield;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)